plugins/catalog/src/components/AboutCard/AboutCard.test.tsx

```diff
@@ -708,8 +708,8 @@ describe('<AboutCard />', () => {
       },
     );
 
-    expect(screen.getByText('View TechDocs')).toBeVisible();
-    expect(screen.getByText('View TechDocs').closest('a')).toBeNull();
+    // expect(screen.getByText('View TechDocs')).toBeInTheDocument();
+    expect(screen.queryByText('View TechDocs')).toBeNull();
```

Comment:
this should be

```suggestion
    expect(screen.getByText('View TechDocs')).toBeDisabled();
```

but you need to bind the route so that the button is rendered.

Also I think it is worth specifying this in the test: `renders disabled techdocs link when no docs exist and route is bound`